main.c

```diff
@@ -561,6 +790,27 @@ int main(int argc,char **argv)
 				else
 					e_additional();
 			}
+			else if (*arg == 'O') {
+				if (argc--)
+					outfilekeys = *argv++;
+				else
+					e_additional();
+			}
+			else if (*arg == 'i') {
+				char * filepath = NULL;
+				char * hostname = NULL;
+				if (argc--) {
+					filepath = *argv++;
+					if (argc--) {
+						hostname = *argv++;
+						return parseandcreate(filepath, hostname);
```

Comment:
should be restructured to be called after argument processing. as it is now, it breaks `--help` argument after itself

Reply:
also, make both arguments optional as specified earlier